src/song.cpp

```diff
@@ -160,6 +167,13 @@ void		Song::init()
 		year = "";
 	}
 
+	/* original year from original date */
```

Comment:
In case `originalyear` doesn't exist, should it be copied from `year` so that it will still display correctly?